onnxruntime/core/framework/session_state.h

```diff
@@ -298,6 +298,8 @@ class SessionState {
   const InlinedHashSet<NodeIndex>* GetToBeExecutedRange(gsl::span<int const> fetch_mlvalue_idxs) const;
 #endif
 
+  Status PruneRemovableAttributes();
```

Comment:
>tus PruneRemovableAttributes(); [](http://example.com/codeflow?start=5&length=31)

We remove unused constants some place in Session Finalize and log it. Can we do something similar here.
Need more input from others to consider the ramifications.
Should this be made configurable?

Reply:
When removing initializers we found it helpful to log names for resolving customer issues. I think the same would be helpful for the attributes.